portfolio/src/main/webapp/script.js

```diff
@@ -59,4 +82,32 @@ async function loadProjects(){
   for(var i =0; i<projects.length; i++){
       projectList.innerHTML += "<li> <a target= \"_blank\" href=\""+projects[i].url+"\">"+projects[i].name;
   }
-}
\ No newline at end of file
+}
+
+function requestTranslation() {
+        const text = document.querySelectorAll('.txt');
+        const languageCode = document.getElementById('language').value;
+        console.log(text);
+        
+
+        
+        for(var i = 0; i<text.length; i++){
+            console.log(text[i].innerText);
+            console.log(languageCode);
+            const responseContainer = text[i];
+            const params = new URLSearchParams();
+            params.append('text', text[i].innerText);
+            params.append('languageCode', languageCode);
+            fetch('/translate', {
+          method: 'POST',
```

Comment:
indentation 🙃

Reply:
Ah yes, this was a nightmare thing, I'll get on that 😅